wgsl/index.bs

```diff
@@ -579,10 +579,17 @@ When unclear from context, this specification indicates
 whether failure to meet a particular requirement
 results in a shader-creation, pipeline-creation, or dynamic error.
 
-The WebGPU specification describes the consequences of each kind of error.  In particular:
-* Detectable errors [=behavioral requirement|will=] [=triggered|trigger=] a [=diagnostic=].
+The consequences of an error are as follows:
 * A WGSL program with a [=shader-creation error=] or [=pipeline-creation error=] error will not be incorporated
     into a [=pipeline=] and hence will not be executed.
+* Detectable errors [=behavioral requirement|will=] [=triggered|trigger=] a [=diagnostic=].
+* If a [=dynamic error=] occurs:
+    * Data access [=behavioral requirement|will=] be restricted to:
+        * [=shader stage inputs=],
+        * [=shader stage outputs=],
+        * any part of a [=resource=] bound to a variable in the WGSL program, and
+        * other variables declared in the WGSL program.
+    * Otherwise, the program may not behave as described in the rest of this specification.
```

Comment:
```suggestion
    * Otherwise, the program may not behave as described in the rest of this specification.
        Note: These effects may be non-local.
```